prime-router/docs/api/submissions.yml

```diff
@@ -0,0 +1,739 @@
+openapi: 3.0.2
+info:
+  title: Prime ReportStream
+  description: A router of public health data from multiple senders and receivers
+  contact:
+    name: USDS at Centers for Disease Control and Prevention
+    url: https://reportstream.cdc.gov
+    email: reportstream@cdc.gov
+  version: 0.2.0-oas3
+paths:
+
+  # The submissions end-point handles receive actions.
```

Comment:
I am thinking we should remove this comments once we update the spec to show this info.